script/Ciel/Day 7/07_01_CIEL07_1B.txt

```diff
@@ -104,30 +103,31 @@ Weren't you the one who said that cool line about leaving the city after the vam
 [sha:5c6d5310684d37fd6fe9e69d87c356318a4569fe]{
 -- Page 39, Offset 14902.
 -- 「おお。そんな話もありましたね。
+// He didn't say that last line out loud? Does the mind reading come with the mind erasing?
```

Comment:
This happens all the time in Tsukihime; where the prose says they say something, but there's not actual dialogue.